### PR TITLE
font update, content addition for current efforts, reform opps, and FAQs tabs, and accordion redesign

### PR DESCRIPTION
**Pull request by kccrtv**
# Description and Motivation
this PR replaces the Taviraj H1 font with DM Sans for consistency across the platform and ensures the submenu is displayed on all breakpoints. it adds content to the Current Efforts and Reform Opportunities tabs, as well as the FAQs tab. additionally, the FAQ accordions have been redesigned for a more user-friendly experience.

## Has this been tested? How?
tests passing

## Screenshots (if appropriate)
Screenshot 2024-08-20 at 2 01 00 PM
Screenshot 2024-08-20 at 2 00 24 PM
Screenshot 2024-08-20 at 2 00 30 PM

## Types of changes
- New content or feature
- Refactor / chore

## New frontend preview link is below in the Netlify comment 😎
add `/policy` at the end of the URL to preview

### CODE REVIEW

File: frontend/src/pages/Policy/policyContent/FaqsContent.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+			<>
+				The integration of gun violence data aims to seamlessly incorporate comprehensive statistics related to gun-related injuries into the Health Equity Tracker platform. This initiative enables users to explore and understand the full spectrum of gun-related incidents and their impacts.
+			</>
+		),
+	},
+	{
+		question: 'What is the main source of the gun violence data?',
+		answer: (
+			<>
+				The primary source of our gun violence data is the CDC's WISQARS dataset, which offers a wide range of information on gun-related injuries, including both fatal and non-fatal incidents. This dataset helps provide a holistic perspective on the effects of gun-related violence.
```
Comment (benhammondmusic) [inline frontend/src/pages/Policy/policyContent/FaqsContent.tsx:22]: remove `non-fatal`

File: frontend/src/pages/Policy/policyContent/FaqsContent.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+			</>
+		),
+	},
+	{
+		question: 'What types of injuries are covered in the dataset?',
+		answer: (
+			<>
+				The dataset includes data on both fatal and non-fatal injuries. Fatal injuries are further categorized into homicide, legal intervention, and suicide.
```
Comment (benhammondmusic) [inline frontend/src/pages/Policy/policyContent/FaqsContent.tsx:30]: remove `non-fatal`

File: frontend/src/pages/Policy/policyContent/FaqsContent.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+			</>
+		),
+	},
+	{
+		question: 'At what geographic levels is the gun violence data available?',
+		answer: (
+			<>
+				<strong>Fatal Injuries:</strong> Data is available at both national and state levels.<br/>
+				<strong>Non-fatal Injuries:</strong> Data is currently available at the national level.
```
Comment (benhammondmusic) [inline frontend/src/pages/Policy/policyContent/FaqsContent.tsx:39]: remove

File: frontend/src/pages/Policy/policyContent/FaqsContent.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+	},
+	{
+		question: 'What demographic details can users filter by in the dataset?',
+		answer: (
+			<>
+				Users can filter the data by:<br/>
+				<strong>Age Groups:</strong> Available in 5-year increments and single-year groups (ranging from 0 to 85+ years, including unknown ages).<br/>
+				<strong>Sex:</strong> Data can be filtered for male and female demographics.<br/>
+				<strong>Race and Ethnicity (for fatal injuries only):</strong> Options include White, Black, American Indian/Alaska Native, Asian, Hawaiian Native/Pacific Islander, more than one race, unknown race, and Hispanic.
+			</>
```
Comment (benhammondmusic) [inline frontend/src/pages/Policy/policyContent/FaqsContent.tsx:59]: You can add `Urbanicity` for Black Men Homicides

File: frontend/src/pages/Policy/policyContent/FaqsContent.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+				Youth-related violence is highlighted by categorizing data into specific age groups relevant to youth (0 - 25 years) and race groups, helping users focus on incidents involving younger populations.
+			</>
+		),
+	},
+	{
+		question: 'What can we expect from the instructional video series?',
+		answer: (
+			<>
+				The upcoming instructional video series will provide clear, concise guidance on how to effectively use the Health Equity Tracker. The videos will be short and digestible, focusing on utilizing gun violence data to demonstrate how to navigate the platform efficiently.
+			</>
```
Comment (benhammondmusic) [inline frontend/src/pages/Policy/policyContent/FaqsContent.tsx:75]: should we link people to the tik tok here for the time being or the youtube?

**Review by benhammondmusic — APPROVED** [review]
great! please make the few changes I noted below before merging. Thanks Kim!

File: frontend/src/pages/Policy/policySections/FaqsTab.tsx [hunk 6]
```diff
@@ -1,12 +1,62 @@
...
+				<div className='p-4'>
+					{policyFaqs.map((faq, index) => (
+						<Accordion
+							key={faq.question}
+							expanded={expanded === `panel${index}`}
+							onChange={handleChange(`panel${index}`)}
+							sx={{
+								border: 'none',
+								boxShadow: 'none',
+								'&::before': {
```
Comment (eriwarr) [inline frontend/src/pages/Policy/policySections/FaqsTab.tsx:31]: Nice!

**Review by eriwarr — APPROVED** [review]
Looks great!

**Comment by eriwarr** [comment]
Screenshot 2024-08-21 at 9 17 16 AM

This already looks great, but I think the section would look better if the cards were aligned with the header.

**Comment by eriwarr** [comment]
Screenshot 2024-08-21 at 9 15 59 AM


The DeKalb County section doesn’t have the same spacing between the header and the text as the other sections.

**Pull request merged**